sympy/concrete/expr_with_limits.py

```diff
@@ -25,12 +26,12 @@ def _process_limits(*symbols):
     limits = []
     orientation = 1
     for V in symbols:
-        if isinstance(V, Symbol):
+        if isinstance(V, (Symbol, Idx)):
```

Comment:
> Well, if i contains information about the range, what's the problem? The way Sum usually works requires to specify a range, because Symbol does not contain that information, while Idx may contain it.

That is True. We can allow something like this ``Sum(A[i], i)`` but something like this
should also be allowed ``Sum(A[i], (i, 1, 4)``.